remoteappmanager/docker/container.py

```diff
@@ -32,12 +32,22 @@ class Container(HasTraits):
     #: the id that will go in the URL of the container
     url_id = Unicode()
 
+    #: The absolute base prefix path for the container.
+    base_urlpath = Unicode(None, allow_none=True)
```

Comment:
If `absurlpath` raises an Error when base_urlpath is None, why do we allow_none here?